core/src/queue.rs

```diff
@@ -695,13 +739,9 @@ mod tests {
         push_txs_handle.join().unwrap();
         get_txs_handle.join().unwrap();
 
-        // Last update for queue to drop invalid txs.
-        let _unused = queue.get_transactions_for_block(&wsv);
-
         // Validate the queue state.
         let array_queue: Vec<_> = core::iter::repeat_with(|| queue.queue.pop())
```

Comment:
`from_fn` again

Reply:
yeap, good suggestion, i was unaware of this function!